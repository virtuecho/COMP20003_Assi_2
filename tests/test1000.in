Summerholm
Pinevale
Carstairs
Mount Eliza
Carlton
Waranga Shores
Hall
Tooloon
Rocky Point
Owen
Lower Mount Walker
Bohena Creek
Roxby Downs
Horsnell Gully
Faulconbridge
Kangaroo Point
Kamarooka
Wonderland
Stratheden
Spotswood
Tenterden
Flinders
Beeron
Springfield
Corop
Electra
Moolerr
Lyneham
Far Far Away
Nashua
Premaydena
Shoal Bay
Camden Head
Wagin
Kensington
Springfield
Hopeland
Dee
Boosey
Wheeo
Pasadena
Kevington
Kuraby
Deepdene
Nelligen
Mount Macarthur
Algorithms & Data Structure
Merlwood
Myrtle Mountain
Port Julia
Wirrimbi
Donald Creek
Tenterfield
Commissioners Creek
Manangatang
Nearum
New York
Blackbutt
Gosforth
Coffs Harbour
Mount Mort
Reedy Dam
Never Ending
Wybung
Gobarralong
Richmond Lowlands
Nyarrin
Evanslea
Balliang
Fadden
Lynton
Hoddys Well
Seddon
Bungundarra
Woodpark
Recherche
Carlton River
Wolvi
Binary Search Tree
Pyengana
Mount Barker Junction
Littlehampton
Gilston
Cuprona
Broadwater
Lisle
Ascot Vale
Wilsons Pocket
Argyll
Richmond
Port Bonython
