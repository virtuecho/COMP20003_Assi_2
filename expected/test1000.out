Summerholm -->
Rec ID1774, Suburb Code32688, Suburb NameSummerholm, Census Year2021, State Code3, State NameQueensland, Local Government Code34580, Local Govrnment NameLockyer Valley, Latitude-27.6090949, Longitude152.4603634
Pinevale -->
Rec ID1933, Suburb Code32327, Suburb NamePinevale, Census Year2021, State Code3, State NameQueensland, Local Government Code34770, Local Govrnment NameMackay, Latitude-21.3300276, Longitude148.8191851
Carstairs -->
Rec ID5760, Suburb Code30554, Suburb NameCarstairs, Census Year2021, State Code3, State NameQueensland, Local Government Code31900, Local Govrnment NameBurdekin, Latitude-19.6510650, Longitude147.4545719
Mount Eliza -->
Rec ID6536, Suburb Code21793, Suburb NameMount Eliza, Census Year2021, State Code2, State NameVictoria, Local Government Code25340, Local Govrnment NameMornington Peninsula, Latitude-38.1949868, Longitude145.0928932
Carlton -->
Rec ID9773, Suburb Code20495, Suburb NameCarlton, Census Year2021, State Code2, State NameVictoria, Local Government Code24600, Local Govrnment NameMelbourne, Latitude-37.8004392, Longitude144.9680900
Rec ID11627, Suburb Code10818, Suburb NameCarlton, Census Year2021, State Code1, State NameNew South Wales, Local Government Code10500, 12930, Local Govrnment NameBayside, Georges River, Latitude-33.9710214, Longitude151.1213916
Rec ID14539, Suburb Code60099, Suburb NameCarlton, Census Year2021, State Code6, State NameTasmania, Local Government Code64810, Local Govrnment NameSorell, Latitude-42.8609790, Longitude147.6536575
Waranga Shores -->
Rec ID2865, Suburb Code22687, Suburb NameWaranga Shores, Census Year2021, State Code2, State NameVictoria, Local Government Code21370, Local Govrnment NameCampaspe, Latitude-36.5480568, Longitude145.0912123
Hall -->
Rec ID1919, Suburb Code80070, Suburb NameHall, Census Year2021, State Code8, State NameAustralian Capital Territory, Local Government Code89399, Local Govrnment NameUnincorporated ACT, Latitude-35.1702040, Longitude149.0669176
Tooloon -->
Rec ID5210, Suburb Code13923, Suburb NameTooloon, Census Year2021, State Code1, State NameNew South Wales, Local Government Code12150, Local Govrnment NameCoonamble, Latitude-30.9998580, Longitude148.1580765
Rocky Point -->
Rec ID1174, Suburb Code32458, Suburb NameRocky Point, Census Year2021, State Code3, State NameQueensland, Local Government Code37300, Local Govrnment NameWeipa, Latitude-12.6238407, Longitude141.8853773
Rec ID10061, Suburb Code41271, Suburb NameRocky Point, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44210, Local Govrnment NameMid Murray, Latitude-34.8541627, Longitude139.4428212
Owen -->
Rec ID8274, Suburb Code41085, Suburb NameOwen, Census Year2021, State Code4, State NameSouth Australia, Local Government Code48130, Local Govrnment NameWakefield, Latitude-34.2504510, Longitude138.5232850
Lower Mount Walker -->
Rec ID9897, Suburb Code31693, Suburb NameLower Mount Walker, Census Year2021, State Code3, State NameQueensland, Local Government Code33960, 36510, Local Govrnment NameIpswich, Scenic Rim, Latitude-27.7214865, Longitude152.5360259
Bohena Creek -->
Rec ID9359, Suburb Code10443, Suburb NameBohena Creek, Census Year2021, State Code1, State NameNew South Wales, Local Government Code15750, Local Govrnment NameNarrabri, Latitude-30.4077098, Longitude149.6322896
Roxby Downs -->
Rec ID5041, Suburb Code41281, Suburb NameRoxby Downs, Census Year2021, State Code4, State NameSouth Australia, Local Government Code46970, Local Govrnment NameRoxby Downs, Latitude-30.5638601, Longitude136.9003411
Horsnell Gully -->
Rec ID9961, Suburb Code40609, Suburb NameHorsnell Gully, Census Year2021, State Code4, State NameSouth Australia, Local Government Code40120, Local Govrnment NameAdelaide Hills, Latitude-34.9351043, Longitude138.7044330
Faulconbridge -->
Rec ID5819, Suburb Code11499, Suburb NameFaulconbridge, Census Year2021, State Code1, State NameNew South Wales, Local Government Code10900, Local Govrnment NameBlue Mountains, Latitude-33.6853210, Longitude150.5510326
Kangaroo Point -->
Rec ID9393, Suburb Code12068, Suburb NameKangaroo Point, Census Year2021, State Code1, State NameNew South Wales, Local Government Code17150, Local Govrnment NameSutherland Shire, Latitude-34.0028078, Longitude151.0958939
Kamarooka -->
Rec ID4704, Suburb Code21289, Suburb NameKamarooka, Census Year2021, State Code2, State NameVictoria, Local Government Code22620, 23940, Local Govrnment NameGreater Bendigo, Loddon, Latitude-36.4968140, Longitude144.3718407
Wonderland -->
Stratheden -->
Rec ID8093, Suburb Code13689, Suburb NameStratheden, Census Year2021, State Code1, State NameNew South Wales, Local Government Code16610, Local Govrnment NameRichmond Valley, Latitude-28.7505048, Longitude152.9468209
Spotswood -->
Rec ID6890, Suburb Code22319, Suburb NameSpotswood, Census Year2021, State Code2, State NameVictoria, Local Government Code23110, Local Govrnment NameHobsons Bay, Latitude-37.8299577, Longitude144.8879621
Tenterden -->
Rec ID5651, Suburb Code13806, Suburb NameTenterden, Census Year2021, State Code1, State NameNew South Wales, Local Government Code10180, Local Govrnment NameArmidale Regional, Latitude-30.1133044, Longitude151.4153636
Flinders -->
Rec ID5051, Suburb Code11537, Suburb NameFlinders, Census Year2021, State Code1, State NameNew South Wales, Local Government Code16900, Local Govrnment NameShellharbour, Latitude-34.5807832, Longitude150.8460702
Beeron -->
Rec ID3014, Suburb Code30204, Suburb NameBeeron, Census Year2021, State Code3, State NameQueensland, Local Government Code35760, Local Govrnment NameNorth Burnett, Latitude-25.8924272, Longitude151.2726336
Springfield -->
Rec ID1576, Suburb Code22325, Suburb NameSpringfield, Census Year2021, State Code2, State NameVictoria, Local Government Code24130, Local Govrnment NameMacedon Ranges, Latitude-37.3313968, Longitude144.8202109
Rec ID2273, Suburb Code51399, Suburb NameSpringfield, Census Year2021, State Code5, State NameWestern Australia, Local Government Code54060, Local Govrnment NameIrwin, Latitude-29.2878607, Longitude114.9540971
Rec ID3201, Suburb Code22324, Suburb NameSpringfield, Census Year2021, State Code2, State NameVictoria, Local Government Code21270, Local Govrnment NameBuloke, Latitude-35.5747195, Longitude143.1143155
Rec ID6636, Suburb Code13640, Suburb NameSpringfield, Census Year2021, State Code1, State NameNew South Wales, Local Government Code17040, Local Govrnment NameSnowy Monaro Regional, Latitude-36.5324737, Longitude149.1239824
Rec ID7655, Suburb Code32627, Suburb NameSpringfield, Census Year2021, State Code3, State NameQueensland, Local Government Code33960, Local Govrnment NameIpswich, Latitude-27.6524455, Longitude152.9101846
Rec ID7871, Suburb Code32628, Suburb NameSpringfield, Census Year2021, State Code3, State NameQueensland, Local Government Code34880, Local Govrnment NameMareeba, Latitude-17.9323097, Longitude144.5156938
Rec ID9592, Suburb Code60630, Suburb NameSpringfield, Census Year2021, State Code6, State NameTasmania, Local Government Code61810, 64010, Local Govrnment NameDorset, Launceston, Latitude-41.2472038, Longitude147.4627848
Rec ID10085, Suburb Code13639, Suburb NameSpringfield, Census Year2021, State Code1, State NameNew South Wales, Local Government Code11650, Local Govrnment NameCentral Coast, Latitude-33.4281967, Longitude151.3727974
Rec ID12029, Suburb Code41363, Suburb NameSpringfield, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44340, Local Govrnment NameMitcham, Latitude-34.9781835, Longitude138.6324971
Corop -->
Rec ID8471, Suburb Code20645, Suburb NameCorop, Census Year2021, State Code2, State NameVictoria, Local Government Code21370, Local Govrnment NameCampaspe, Latitude-36.4613514, Longitude144.7802750
Electra -->
Rec ID9051, Suburb Code30957, Suburb NameElectra, Census Year2021, State Code3, State NameQueensland, Local Government Code31820, Local Govrnment NameBundaberg, Latitude-24.9969550, Longitude152.1153589
Moolerr -->
Rec ID4140, Suburb Code21738, Suburb NameMoolerr, Census Year2021, State Code2, State NameVictoria, Local Government Code25810, Local Govrnment NameNorthern Grampians, Latitude-36.6436595, Longitude143.1717845
Lyneham -->
Rec ID7598, Suburb Code80086, Suburb NameLyneham, Census Year2021, State Code8, State NameAustralian Capital Territory, Local Government Code89399, Local Govrnment NameUnincorporated ACT, Latitude-35.2397954, Longitude149.1307413
Far Far Away -->
Nashua -->
Rec ID10233, Suburb Code12923, Suburb NameNashua, Census Year2021, State Code1, State NameNew South Wales, Local Government Code11350, Local Govrnment NameByron, Latitude-28.7270907, Longitude153.4650835
Premaydena -->
Rec ID9550, Suburb Code60518, Suburb NamePremaydena, Census Year2021, State Code6, State NameTasmania, Local Government Code65210, Local Govrnment NameTasman, Latitude-43.0577293, Longitude147.7584337
Shoal Bay -->
Rec ID7545, Suburb Code70240, Suburb NameShoal Bay, Census Year2021, State Code7, State NameNorthern Territory, Local Government Code72300, Local Govrnment NameLitchfield, Latitude-12.3707577, Longitude131.0504022
Camden Head -->
Rec ID9248, Suburb Code10770, Suburb NameCamden Head, Census Year2021, State Code1, State NameNew South Wales, Local Government Code16380, Local Govrnment NamePort Macquarie-Hastings, Latitude-31.6436536, Longitude152.8309533
Wagin -->
Rec ID2072, Suburb Code51501, Suburb NameWagin, Census Year2021, State Code5, State NameWestern Australia, Local Government Code58610, Local Govrnment NameWagin, Latitude-33.3017289, Longitude117.3453958
Kensington -->
Rec ID1117, Suburb Code21327, Suburb NameKensington, Census Year2021, State Code2, State NameVictoria, Local Government Code24330, 24600, 25060, Local Govrnment NameMaribyrnong, Melbourne, Moonee Valley, Latitude-37.7943466, Longitude144.9270189
Rec ID1986, Suburb Code31507, Suburb NameKensington, Census Year2021, State Code3, State NameQueensland, Local Government Code31820, Local Govrnment NameBundaberg, Latitude-24.9170988, Longitude152.3182806
Rec ID6206, Suburb Code50752, Suburb NameKensington, Census Year2021, State Code5, State NameWestern Australia, Local Government Code57840, 58510, Local Govrnment NameSouth Perth, Victoria Park, Latitude-31.9856189, Longitude115.8839737
Rec ID7720, Suburb Code12107, Suburb NameKensington, Census Year2021, State Code1, State NameNew South Wales, Local Government Code16550, Local Govrnment NameRandwick, Latitude-33.9134157, Longitude151.2207450
Rec ID10486, Suburb Code40677, Suburb NameKensington, Census Year2021, State Code4, State NameSouth Australia, Local Government Code45290, Local Govrnment NameNorwood Payneham and St Peters, Latitude-34.9237458, Longitude138.6456183
Springfield -->
Rec ID1576, Suburb Code22325, Suburb NameSpringfield, Census Year2021, State Code2, State NameVictoria, Local Government Code24130, Local Govrnment NameMacedon Ranges, Latitude-37.3313968, Longitude144.8202109
Rec ID2273, Suburb Code51399, Suburb NameSpringfield, Census Year2021, State Code5, State NameWestern Australia, Local Government Code54060, Local Govrnment NameIrwin, Latitude-29.2878607, Longitude114.9540971
Rec ID3201, Suburb Code22324, Suburb NameSpringfield, Census Year2021, State Code2, State NameVictoria, Local Government Code21270, Local Govrnment NameBuloke, Latitude-35.5747195, Longitude143.1143155
Rec ID6636, Suburb Code13640, Suburb NameSpringfield, Census Year2021, State Code1, State NameNew South Wales, Local Government Code17040, Local Govrnment NameSnowy Monaro Regional, Latitude-36.5324737, Longitude149.1239824
Rec ID7655, Suburb Code32627, Suburb NameSpringfield, Census Year2021, State Code3, State NameQueensland, Local Government Code33960, Local Govrnment NameIpswich, Latitude-27.6524455, Longitude152.9101846
Rec ID7871, Suburb Code32628, Suburb NameSpringfield, Census Year2021, State Code3, State NameQueensland, Local Government Code34880, Local Govrnment NameMareeba, Latitude-17.9323097, Longitude144.5156938
Rec ID9592, Suburb Code60630, Suburb NameSpringfield, Census Year2021, State Code6, State NameTasmania, Local Government Code61810, 64010, Local Govrnment NameDorset, Launceston, Latitude-41.2472038, Longitude147.4627848
Rec ID10085, Suburb Code13639, Suburb NameSpringfield, Census Year2021, State Code1, State NameNew South Wales, Local Government Code11650, Local Govrnment NameCentral Coast, Latitude-33.4281967, Longitude151.3727974
Rec ID12029, Suburb Code41363, Suburb NameSpringfield, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44340, Local Govrnment NameMitcham, Latitude-34.9781835, Longitude138.6324971
Hopeland -->
Rec ID5839, Suburb Code31358, Suburb NameHopeland, Census Year2021, State Code3, State NameQueensland, Local Government Code37310, Local Govrnment NameWestern Downs, Latitude-26.8874670, Longitude150.6627219
Dee -->
Rec ID235, Suburb Code60146, Suburb NameDee, Census Year2021, State Code6, State NameTasmania, Local Government Code61010, Local Govrnment NameCentral Highlands, Latitude-42.2518179, Longitude146.5926802
Boosey -->
Rec ID4767, Suburb Code20301, Suburb NameBoosey, Census Year2021, State Code2, State NameVictoria, Local Government Code24900, Local Govrnment NameMoira, Latitude-36.0504864, Longitude145.7916177
Wheeo -->
Rec ID5770, Suburb Code14287, Suburb NameWheeo, Census Year2021, State Code1, State NameNew South Wales, Local Government Code17640, Local Govrnment NameUpper Lachlan Shire, Latitude-34.5133322, Longitude149.2786628
Pasadena -->
Rec ID1964, Suburb Code41120, Suburb NamePasadena, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44340, Local Govrnment NameMitcham, Latitude-35.0027195, Longitude138.5886217
Kevington -->
Rec ID8568, Suburb Code21335, Suburb NameKevington, Census Year2021, State Code2, State NameVictoria, Local Government Code24250, Local Govrnment NameMansfield, Latitude-37.3601737, Longitude146.1693989
Kuraby -->
Rec ID686, Suburb Code31588, Suburb NameKuraby, Census Year2021, State Code3, State NameQueensland, Local Government Code31000, Local Govrnment NameBrisbane, Latitude-27.6068481, Longitude153.0936220
Deepdene -->
Rec ID6738, Suburb Code20728, Suburb NameDeepdene, Census Year2021, State Code2, State NameVictoria, Local Government Code21110, Local Govrnment NameBoroondara, Latitude-37.8116220, Longitude145.0659660
Nelligen -->
Rec ID3320, Suburb Code12929, Suburb NameNelligen, Census Year2021, State Code1, State NameNew South Wales, Local Government Code12750, Local Govrnment NameEurobodalla, Latitude-35.6668681, Longitude150.1006544
Mount Macarthur -->
Rec ID771, Suburb Code31988, Suburb NameMount Macarthur, Census Year2021, State Code3, State NameQueensland, Local Government Code32270, Local Govrnment NameCentral Highlands, Latitude-22.8925976, Longitude148.2061897
Algorithms & Data Structure -->
Merlwood -->
Rec ID10052, Suburb Code31810, Suburb NameMerlwood, Census Year2021, State Code3, State NameQueensland, Local Government Code36630, Local Govrnment NameSouth Burnett, Latitude-26.1526346, Longitude151.8790205
Myrtle Mountain -->
Rec ID6634, Suburb Code12891, Suburb NameMyrtle Mountain, Census Year2021, State Code1, State NameNew South Wales, Local Government Code10550, Local Govrnment NameBega Valley, Latitude-36.8512682, Longitude149.6873340
Port Julia -->
Rec ID11059, Suburb Code41199, Suburb NamePort Julia, Census Year2021, State Code4, State NameSouth Australia, Local Government Code48830, Local Govrnment NameYorke Peninsula, Latitude-34.6475106, Longitude137.8685300
Wirrimbi -->
Rec ID9699, Suburb Code14359, Suburb NameWirrimbi, Census Year2021, State Code1, State NameNew South Wales, Local Government Code15700, Local Govrnment NameNambucca Valley, Latitude-30.6709463, Longitude152.9259862
Donald Creek -->
Rec ID11365, Suburb Code11273, Suburb NameDonald Creek, Census Year2021, State Code1, State NameNew South Wales, Local Government Code10180, Local Govrnment NameArmidale Regional, Latitude-30.4263661, Longitude151.7613923
Tenterfield -->
Rec ID8015, Suburb Code13807, Suburb NameTenterfield, Census Year2021, State Code1, State NameNew South Wales, Local Government Code17400, Local Govrnment NameTenterfield, Latitude-29.0807666, Longitude152.0344543
Commissioners Creek -->
Rec ID2587, Suburb Code10995, Suburb NameCommissioners Creek, Census Year2021, State Code1, State NameNew South Wales, Local Government Code17550, Local Govrnment NameTweed, Latitude-28.4989104, Longitude153.3321721
Manangatang -->
Rec ID779, Suburb Code21589, Suburb NameManangatang, Census Year2021, State Code2, State NameVictoria, Local Government Code26610, Local Govrnment NameSwan Hill, Latitude-35.0527593, Longitude142.9002848
Nearum -->
Rec ID11021, Suburb Code32111, Suburb NameNearum, Census Year2021, State Code3, State NameQueensland, Local Government Code31820, Local Govrnment NameBundaberg, Latitude-25.0708780, Longitude151.8093443
New York -->
Blackbutt -->
Rec ID9281, Suburb Code10390, Suburb NameBlackbutt, Census Year2021, State Code1, State NameNew South Wales, Local Government Code16900, Local Govrnment NameShellharbour, Latitude-34.5710125, Longitude150.8399417
Gosforth -->
Rec ID9368, Suburb Code11729, Suburb NameGosforth, Census Year2021, State Code1, State NameNew South Wales, Local Government Code15050, Local Govrnment NameMaitland, Latitude-32.6523767, Longitude151.4897357
Coffs Harbour -->
Rec ID9232, Suburb Code10959, Suburb NameCoffs Harbour, Census Year2021, State Code1, State NameNew South Wales, Local Government Code11800, Local Govrnment NameCoffs Harbour, Latitude-30.2982358, Longitude153.1076847
Mount Mort -->
Rec ID8659, Suburb Code32003, Suburb NameMount Mort, Census Year2021, State Code3, State NameQueensland, Local Government Code33960, Local Govrnment NameIpswich, Latitude-27.7765559, Longitude152.4330202
Reedy Dam -->
Rec ID770, Suburb Code22156, Suburb NameReedy Dam, Census Year2021, State Code2, State NameVictoria, Local Government Code21270, Local Govrnment NameBuloke, Latitude-35.9579007, Longitude142.6452758
Never Ending -->
Wybung -->
Rec ID9223, Suburb Code14455, Suburb NameWybung, Census Year2021, State Code1, State NameNew South Wales, Local Government Code11650, Local Govrnment NameCentral Coast, Latitude-33.1923388, Longitude151.6045639
Gobarralong -->
Rec ID4773, Suburb Code11692, Suburb NameGobarralong, Census Year2021, State Code1, State NameNew South Wales, Local Government Code12160, Local Govrnment NameCootamundra-Gundagai Regional, Latitude-34.9519254, Longitude148.3556424
Richmond Lowlands -->
Rec ID8892, Suburb Code13377, Suburb NameRichmond Lowlands, Census Year2021, State Code1, State NameNew South Wales, Local Government Code13800, Local Govrnment NameHawkesbury, Latitude-33.5789799, Longitude150.7529625
Nyarrin -->
Rec ID4098, Suburb Code21994, Suburb NameNyarrin, Census Year2021, State Code2, State NameVictoria, Local Government Code21270, Local Govrnment NameBuloke, Latitude-35.3865148, Longitude142.6707409
Evanslea -->
Rec ID713, Suburb Code31008, Suburb NameEvanslea, Census Year2021, State Code3, State NameQueensland, Local Government Code36910, Local Govrnment NameToowoomba, Latitude-27.5323946, Longitude151.5191670
Balliang -->
Rec ID2039, Suburb Code20115, Suburb NameBalliang, Census Year2021, State Code2, State NameVictoria, Local Government Code22750, 25150, 27260, Local Govrnment NameGreater Geelong, Moorabool, Wyndham, Latitude-37.8184284, Longitude144.3450237
Fadden -->
Rec ID4819, Suburb Code80051, Suburb NameFadden, Census Year2021, State Code8, State NameAustralian Capital Territory, Local Government Code89399, Local Govrnment NameUnincorporated ACT, Latitude-35.4020188, Longitude149.1176950
Lynton -->
Rec ID9133, Suburb Code40795, Suburb NameLynton, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44340, Local Govrnment NameMitcham, Latitude-34.9973223, Longitude138.6073899
Hoddys Well -->
Rec ID9448, Suburb Code50646, Suburb NameHoddys Well, Census Year2021, State Code5, State NameWestern Australia, Local Government Code58330, Local Govrnment NameToodyay, Latitude-31.6457252, Longitude116.4584474
Seddon -->
Rec ID5446, Suburb Code22256, Suburb NameSeddon, Census Year2021, State Code2, State NameVictoria, Local Government Code24330, Local Govrnment NameMaribyrnong, Latitude-37.8066284, Longitude144.8917067
Rec ID8161, Suburb Code41325, Suburb NameSeddon, Census Year2021, State Code4, State NameSouth Australia, Local Government Code42750, Local Govrnment NameKangaroo Island, Latitude-35.8589260, Longitude137.2800867
Bungundarra -->
Rec ID4536, Suburb Code30434, Suburb NameBungundarra, Census Year2021, State Code3, State NameQueensland, Local Government Code34530, Local Govrnment NameLivingstone, Latitude-23.0488969, Longitude150.6383748
Woodpark -->
Rec ID1890, Suburb Code14408, Suburb NameWoodpark, Census Year2021, State Code1, State NameNew South Wales, Local Government Code12380, Local Govrnment NameCumberland, Latitude-33.8413048, Longitude150.9604336
Recherche -->
Rec ID8382, Suburb Code60539, Suburb NameRecherche, Census Year2021, State Code6, State NameTasmania, Local Government Code63010, Local Govrnment NameHuon Valley, Latitude-43.5337768, Longitude146.8652178
Carlton River -->
Rec ID7502, Suburb Code60100, Suburb NameCarlton River, Census Year2021, State Code6, State NameTasmania, Local Government Code64810, Local Govrnment NameSorell, Latitude-42.8642702, Longitude147.7112742
Wolvi -->
Rec ID10012, Suburb Code33112, Suburb NameWolvi, Census Year2021, State Code3, State NameQueensland, Local Government Code33620, Local Govrnment NameGympie, Latitude-26.1502807, Longitude152.8253867
Binary Search Tree -->
Pyengana -->
Rec ID9466, Suburb Code60527, Suburb NamePyengana, Census Year2021, State Code6, State NameTasmania, Local Government Code60210, Local Govrnment NameBreak O'Day, Latitude-41.3002816, Longitude147.9487567
Mount Barker Junction -->
Rec ID2426, Suburb Code40933, Suburb NameMount Barker Junction, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44550, Local Govrnment NameMount Barker, Latitude-35.0253491, Longitude138.8687596
Littlehampton -->
Rec ID3233, Suburb Code40770, Suburb NameLittlehampton, Census Year2021, State Code4, State NameSouth Australia, Local Government Code44550, Local Govrnment NameMount Barker, Latitude-35.0399800, Longitude138.8668249
Gilston -->
Rec ID8636, Suburb Code31123, Suburb NameGilston, Census Year2021, State Code3, State NameQueensland, Local Government Code33430, Local Govrnment NameGold Coast, Latitude-28.0296606, Longitude153.3046678
Cuprona -->
Rec ID6886, Suburb Code60141, Suburb NameCuprona, Census Year2021, State Code6, State NameTasmania, Local Government Code60810, Local Govrnment NameCentral Coast, Latitude-41.1269384, Longitude145.9705911
Broadwater -->
Rec ID2709, Suburb Code10577, Suburb NameBroadwater, Census Year2021, State Code1, State NameNew South Wales, Local Government Code10250, 14850, 16610, Local Govrnment NameBallina, Lismore, Richmond Valley, Latitude-29.0350454, Longitude153.4261282
Lisle -->
Rec ID2251, Suburb Code60343, Suburb NameLisle, Census Year2021, State Code6, State NameTasmania, Local Government Code61810, Local Govrnment NameDorset, Latitude-41.2346437, Longitude147.3270020
Ascot Vale -->
Rec ID960, Suburb Code20075, Suburb NameAscot Vale, Census Year2021, State Code2, State NameVictoria, Local Government Code25060, Local Govrnment NameMoonee Valley, Latitude-37.7770335, Longitude144.9136945
Wilsons Pocket -->
Rec ID2071, Suburb Code33086, Suburb NameWilsons Pocket, Census Year2021, State Code3, State NameQueensland, Local Government Code33620, Local Govrnment NameGympie, Latitude-26.1231096, Longitude152.7999159
Argyll -->
Rec ID4823, Suburb Code30083, Suburb NameArgyll, Census Year2021, State Code3, State NameQueensland, Local Government Code32270, Local Govrnment NameCentral Highlands, Latitude-23.2986730, Longitude147.4795040
Richmond -->
Rec ID577, Suburb Code41256, Suburb NameRichmond, Census Year2021, State Code4, State NameSouth Australia, Local Government Code48410, Local Govrnment NameWest Torrens, Latitude-34.9386376, Longitude138.5621269
Rec ID2856, Suburb Code60551, Suburb NameRichmond, Census Year2021, State Code6, State NameTasmania, Local Government Code61410, Local Govrnment NameClarence, Latitude-42.7387184, Longitude147.4137797
Rec ID8552, Suburb Code32424, Suburb NameRichmond, Census Year2021, State Code3, State NameQueensland, Local Government Code34770, Local Govrnment NameMackay, Latitude-21.0869211, Longitude149.1405356
Rec ID12417, Suburb Code13375, Suburb NameRichmond, Census Year2021, State Code1, State NameNew South Wales, Local Government Code13800, Local Govrnment NameHawkesbury, Latitude-33.6048433, Longitude150.7574251
Rec ID12852, Suburb Code32425, Suburb NameRichmond, Census Year2021, State Code3, State NameQueensland, Local Government Code36300, Local Govrnment NameRichmond, Latitude-20.7622455, Longitude143.2365906
Rec ID14926, Suburb Code22170, Suburb NameRichmond, Census Year2021, State Code2, State NameVictoria, Local Government Code27350, Local Govrnment NameYarra, Latitude-37.8202671, Longitude145.0024290
Port Bonython -->
Rec ID1721, Suburb Code41191, Suburb NamePort Bonython, Census Year2021, State Code4, State NameSouth Australia, Local Government Code48540, Local Govrnment NameWhyalla, Latitude-32.9710306, Longitude137.7435731
